packages/blade/src/components/ListView/ListViewFilters.web.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react';
+import type { ListViewFilterProps } from './types';
+import { ListViewFiltersProvider } from './ListViewFiltersContext.web';
+import { makeAnalyticsAttribute } from '~utils/makeAnalyticsAttribute';
+import BaseBox from '~components/Box/BaseBox';
+import { MetaConstants, metaAttribute } from '~utils/metaAttribute';
+import { FilterIcon } from '~components/Icons';
+import { Button } from '~components/Button';
+import { Counter } from '~components/Counter';
+import { Box } from '~components/Box';
+
+const ListViewFilters = ({
+  testID,
+  children,
+  quickFilters,
+  onSearchChange,
+  ...rest
+}: ListViewFilterProps): React.ReactElement => {
+  const [showFilters, setShowFilters] = useState(false);
```

Comment:
Can we not show quickfilters by default? Consumers should have control on wether they want to show by default or not.

Example: Filters are already applied via some controlled state management (eg: query param) and product wants to expand the filters by default